docs/appendices/release-notes/unreleased.rst

```diff
@@ -113,6 +113,7 @@ Fixes
 .. stable branch. You can add a version label (`v/X.Y`) to the pull request for
 .. an automated mergify backport.
 
+- Fixed an issue causing
```

Comment:
This is missing some text?

Reply:
Sorry, something messed up with the master merges.